cpp/ql/src/experimental/Security/CWE/CWE-570/WrongInDetectingAndHandlingMemoryAllocationErrors.ql

```diff
@@ -1,7 +1,7 @@
 /**
  * @name Detect And Handle Memory Allocation Errors
- * @description --::operator new(std::size_t) throws an exception on error, and ::operator new(std::size_t, const std::nothrow_t &) returns zero on error.
- *              --the programmer can get confused when check the error that occurs when allocating memory incorrectly.
+ * @description `operator new` throws an exception on allocation failures, while `operator new(std::nothrow)` returns a null pointer.
+ *              Mixing up these two failure conditions can result in unexpected behavior.
```

Comment:
Are you confident that the newline / indentation here will be displayed correctly in places this description is used?

Reply:
Hm, no. Good point. And it does look like we don't particularly care about long lines in the `@description`. I'll merge that into one line.

Reply:
Fixed in c67ab8f1f0dd170d3b6bbc13f39e2b39e009a346.